packages/ndk-multilib/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://developer.android.com/tools/sdk/ndk/index.html
 TERMUX_PKG_DESCRIPTION="Multilib binaries for cross-compilation"
 TERMUX_PKG_LICENSE="NCSA"
 TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_DEPENDS="libcompiler-rt"
```

Comment:
Can't this be `TERMUX_PKG_BUILD_DEPENDS` instead?

Also would you please move dependency-related definitions just below the source-related definitions (`TERMUX_PKG_SKIP_SRC_EXTRACT` in this case)?

Reply:
OK. I will fix that.

Reply:
I think dependency on libcompiler-rt should be removed. In fact, ndk-multilib should not depend on anything since its source is from NDK. Postinst can take care of symlinks stuff.